Dockerfile.s390x

```diff
@@ -1,7 +1,7 @@
-ARG GOLANG_VERSION=1.19.0-alpine3.15
+ARG GO_VERSION=1.19.0-alpine3.15
```

Comment:
I'm really confused about how this is working, in the other dockerfiles GO_VERSION is just the bare go version, here we have the -alpine suffix on it, and then we add -alpine again on line 4?

Reply:
The GOLANG_VERSION arg is already used in the library/golang image; perhaps there was some interaction with that?

https://github.com/docker-library/golang/blob/master/1.20/alpine3.17/Dockerfile#L13

Reply:
env var refs updated.